host/src/app.c

```diff
@@ -756,7 +756,7 @@ int app_main(int argc, char * argv[])
       const uint64_t delta = now - previousFrameTime;
       if (delta < throttleUs)
       {
-        nsleep((throttleUs - delta) * 1000);
+        usleep(throttleUs - delta);
```

Comment:
`nsleep` should be preferred on code that executes on Windows since it is implemented with `NtDelayExecution` instead of whatever method MinGW uses.

Reply:
I only made this change as the original `nsleep((throttleUs - delta) * 1000);` did not appear to actually be sleeping for some reason (and the rest of `app.c` seemed to use `usleep`).

Any ideas as to why `nsleep` would not work properly in this file? The issue is reproducible yet `nsleep` seems to work fine in `xcb.c`.

Reply:
`nsleep` is part of our `common` lib, we have not really used/tested it under Linux to date, as quantum stated, it's a perf improvement for Windows. It might be bugged.

Reply:
It looks like it was an issue with the types when using `1e9` in `nsleep`. I fixed that and reverted the sleep above to an `nsleep`.